src/shared/contact/delete-note-container.tsx

```diff
@@ -76,7 +77,7 @@ export const DeleteNotePopover = withRouter(
               <Flex justifyContent="flex-start">
                 <Mutation mutation={DELETE_NOTE}>
                   {(
-                    deleteContact: MutationFn<{ id: string }>,
+                    deleteContact: MutationFunction<{ id: string }>,
```

Comment:
need to migrate to `useMutation`

Reply:
Sure, I just temperately migrated these places for making it pass the test after upgrading to apollo client 3. Will fix them one by one later.